src/status_im2/contexts/chat/messages/navigation/view.cljs

```diff
@@ -31,8 +31,8 @@
                                                              {:extrapolateLeft  "clamp"
                                                               :extrapolateRight "extend"})
         translate-animation          (reanimated/interpolate scroll-y
-                                                             [(+ style/navigation-bar-height 25)
-                                                              (+ style/navigation-bar-height 100)]
+                                                             [style/navigation-bar-height
+                                                              (+ style/navigation-bar-height 30)]
```

Comment:
It seems like there is a conflicting change being introduced in a different PR by @ibrkhalil: https://github.com/status-im/status-mobile/pull/17582/files#diff-dcb75a45e3f59f386be45e5a32e867baa9cc961029953cb1fc3d09834b05bb92L34-R34